main.js

```diff
@@ -0,0 +1,7 @@
+import App from './src/app';
+import './src/style.css';
+
+window.addEventListener('DOMContentLoaded', () => {
+  const app = new App(document.querySelector('#app'));
```

Comment:
지금은 크게 상관은 없지만 자바스크립트로 개발을 할 때는 아래와 같이 DOM을 가져올 수 있도록 유틸 함수도 만들어주는 편이에요~
```suggestion
  const app = new App($('#app'));
```

Reply:
아 그럼 $는 보통 document.querySelector로 해주는 편인가요? getElementById도 자주 사용하는 유틸함수명 같은게 있을까요?

Reply:
$는 사실 jQuery에서 제공하는 함수이긴 한데요~
간략하게 표현하기에 좋아서 vanilla javascript로 개발을 하는 경우에도 요렇게 표현해주기도 해요.
getElementById와 같이 특정한 무엇을 이용해서 찾는건 따로 사용하지 않고 내부적으로는 모두 querySelector로 가져오도록 구현하는 편이에요.